api/v3/Case/Getstats.php

```diff
@@ -131,11 +131,11 @@ function _civicrm_api3_case_add_case_category_query_filter($query, array $caseTy
   $caseTypeIds = array_column($caseTypes['values'], 'id');
   $param = ['IN' => $caseTypeIds];
 
-  if ($caseTypes[0] === 'IS NULL') {
+  if ($caseTypeIds[0] === 'IS NULL') {
```

Comment:
The case typeIds might be empty and this will cause a notice.

Why don't we do

```php
if (isset($caseTypeIds[0]) && $caseTypeIds[0] === 'IS NULL') 
```

Also let's move the 
```
  if (empty($caseTypeIds)) {
    return;
  }
```
to immediately after 
```php
$caseTypeIds = array_column($caseTypes['values'], 'id');
```